src/components/PickerMonth.vue

```diff
@@ -181,8 +181,9 @@ export default {
       }
       if (typeof this.disabledDates.from !== 'undefined' && this.disabledDates.from) {
         if (
-          this.disabledDates.from &&
-          (this.utils.getMonth(date) > this.utils.getMonth(this.disabledDates.from) && this.utils.getFullYear(date) >= this.utils.getFullYear(this.disabledDates.from)) ||
+          (this.disabledDates.from &&
+            this.utils.getMonth(date) > this.utils.getMonth(this.disabledDates.from) &&
```

Comment:
What is this change of behaviour for?
In any case I don't think we need that `this.disabledDates.from` as it's handled in the line before.

Reply:
The linter was complaining about mixing the && and || operators like that. I'm pretty sure this doesn't change any functionality?